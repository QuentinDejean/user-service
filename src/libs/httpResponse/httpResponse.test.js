describe('it will work', () => {
  it('yest', () => {
    expect(true).toEqual(true)
  })
})
