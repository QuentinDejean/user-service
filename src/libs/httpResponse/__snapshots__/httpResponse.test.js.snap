// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN httpResponse AND the success method WHEN the method is executed AND all arguments are passed through SHOULD return the correct payload 1`] = `
Object {
  "errors": Object {
    "detail": "An unknown error occured",
    "status": Object {
      "code": 403,
      "message": "forbidden",
    },
  },
  "statusCode": Object {
    "code": 403,
    "message": "forbidden",
  },
}
`;

exports[`GIVEN httpResponse AND the success method WHEN the method is executed AND no arguments are passed through SHOULD return the correct payload 1`] = `
Object {
  "errors": Object {
    "detail": "An unknown error occured",
    "status": 500,
  },
  "statusCode": 500,
}
`;

exports[`GIVEN httpResponse AND the success method WHEN the method is executed AND no message is passed through SHOULD return the correct payload 1`] = `
Object {
  "errors": Object {
    "detail": "An unknown error occured",
    "status": Object {
      "code": 403,
    },
  },
  "statusCode": Object {
    "code": 403,
  },
}
`;

exports[`GIVEN httpResponse AND the success method WHEN the method is executed SHOULD return the correct payload 1`] = `
Object {
  "data": "{\\"message\\":\\"success\\"}",
  "statusCode": 200,
}
`;
